### PR TITLE
Review0 fixes

### PR DESCRIPTION
**Pull request by Sachin-A**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+out = blake2b
+
+blake: $(obj)
+	$(CC) $(obj) -o $(out) $(CFLAGS)
+
+$(ODIR)/%.o: src/%.c
+	$(CC) -c $< -o $@ $(CFLAGS)
+
+clean:
+	rm -f $(wildcard $(ODIR)/*.o) $(wildcard *~) core  $(out).exe
```
Comment (sooryan) [inline Makefile]: You still need to remove $(out) too

File: src/blake2b.c [hunk 2]
```diff
@@ -76,10 +81,10 @@ store32(void* dst, uint32_t w)
  * @param[in]  inc   the increment value
```
Comment (sooryan) [inline src/blake2b.c:78]: @param S -> state

File: src/blake2b.c [hunk 3]
```diff
@@ -155,29 +160,29 @@ F(blake2b_state* state, const uint8_t block[BLAKE2B_BLOCKBYTES])
  *
  * @return     sanity value
```
Comment (sooryan) [inline src/blake2b.c]: no @return  for void

File: src/blake2b.c [hunk 4]
```diff
@@ -206,7 +210,7 @@ blake2b_init(blake2b_state* state, size_t outlen, const void* key, size_t keylen
  *
  * @return     error code
```
Comment (sooryan) [inline src/blake2b.c]: no @return

File: src/blake2b.c [hunk 5]
```diff
@@ -221,19 +225,19 @@ blake2b_update(blake2b_state* state, const void* input_buffer, size_t inlen)
   }
   memcpy(state->buf + state->buflen, in, inlen);
   state->buflen += inlen;
-  return 0;
 }
 
-int
+void
```
Comment (sooryan) [inline src/blake2b.c:230]: No description for this function

File: src/blake2b.c (hunk 4 above)
Comment (Sachin-A) [reply src/blake2b.c]: Done

File: src/blake2b.c (hunk 3 above)
Comment (Sachin-A) [reply src/blake2b.c]: Done

**Pull request merged**